packages/bitcore-node/scripts/test/syncTxnAndCoinsWallets.spec.js

```diff
@@ -0,0 +1,715 @@
+        "chain": "BTC",
+        "mintIndex": 1,
+        "mintTxid": "8af29b05e56294d3fb62b8ac6ba23d01eb44bf438f0bb0a349e0553bc108f05f",
+        "network": "mainnet",
+        "address": "bc1p6d59ukemxzuewmvx983lm8xlvn7gvazppxh425flr63dg5uzpt3qr6yp09",
+        "mintHeight": 784431,
+        "coinbase": false,
+        "value": 36070,
+        "script": "BinData(0,\"USDTaF5bOzC5l22GKeP9nN9k/IZ0QQmvVVE/HqLUU4IK4g==\")",
+        "spentHeight": -2,
+        "wallets": []
+      },
+      {
+        "_id": "64384bb54d124e905184dfe0",
+        "chain": "BTC",
+        "mintIndex": 0,
+        "mintTxid": "8af29b05e56294d3fb62b8ac6ba23d01eb44bf438f0bb0a349e0553bc108f05f",
+        "network": "mainnet",
+        "address": "bc1ptcppl5qc0tchwnqwl2j9nsnh09vtmpc3ng6pzy56aaeq5995ghkq54ke2r",
+        "mintHeight": 784431,
+        "coinbase": false,
+        "value": 1195,
+        "script": "BinData(0,\"USBeAh/QGHrxd0wO+qRZwnd5WL2HEZo0ERKa73IKFLRF7A==\")",
+        "spentHeight": 784431,
+        "wallets": [],
+        "spentTxid": "b387b21dcd4af2b385276ef44a33ef06419b0ab9821d3ec6aacf29b339f5eee0",
+        "sequenceNumber": 4294967293
+      }
+    ]
+  },
+
+  {
+    "_id": "64384bb84d124e905184eaa7",
+    "chain": "BTC",
+    "network": "mainnet",
+    "txid": "cfbfb835e7227608e927edb6ea949e0a563ba463514a26f2a1982e38d4c6699a",
+    "blockHeight": 784431,
+    "blockHash": "0000000000000000000504cebfd5e445001f4769f0ce6607dcd753bd5ee883fd",
+    "blockTime": "2023-04-08T01:35:07.000Z",
+    "blockTimeNormalized": "2023-04-08T01:35:07.000Z",
+    "coinbase": false,
+    "fee": 1484,
+    "size": 94,
+    "locktime": 0,
+    "inputCount": 1,
+    "outputCount": 1,
+    "value": 546,
+    "wallets": [wallets[1]],
+
+    COINS: [
+      {
+        "_id": "64384bb44d124e905184d8f6",
+        "chain": "BTC",
+        "mintIndex": 7,
+        "mintTxid": "8d549b05a91a1852c8152ca7b65fa39654ad87d1e6833e9b0a5ee8897253c6fa",
+        "network": "mainnet",
+        "address": "bc1pzxt0d2p0f8mfglkln3z0kzs5j7wedkwf7dcgvnd7d6pyfnukz5fqcw9mvt",
+        "mintHeight": 784431,
+        "coinbase": false,
+        "value": 2030,
+        "script": "BinData(0,\"USARlvaoL0n2lH7fnET7ChSXnZbZyfNwhk2+boJEz5YVEg==\")",
+        "spentHeight": 784431,
+        "wallets": [wallets[1]],
+        "spentTxid": "cfbfb835e7227608e927edb6ea949e0a563ba463514a26f2a1982e38d4c6699a",
+        "sequenceNumber": 4294967295
+      },
+      {
+        "_id": "64384bb54d124e905184dfaa",
+        "chain": "BTC",
+        "mintIndex": 0,
+        "mintTxid": "cfbfb835e7227608e927edb6ea949e0a563ba463514a26f2a1982e38d4c6699a",
+        "network": "mainnet",
+        "address": "bc1pyymz64g66dqrm2nkvmcswjqhw4c3evjrmdygyqsg0u0dj5rg59msr7j2sk",
+        "mintHeight": 784431,
+        "coinbase": false,
+        "value": 546,
+        "script": "BinData(0,\"USAhNi1VGtNAPap2ZvEHSBd1cRyyQ9tIggIIfx7ZUGihdw==\")",
+        "spentHeight": -2,
+        "wallets": [wallets[1]]
+      }
+    ]
+  }
+]
```

Comment:
Add semicolon. This is partially my bad. When I removed prettier awhile back, things like this stopped getting enforced on commit. We need to fix that before things get too messy.

Reply:
Are you seeing a lint squiggly? If so, which linter is catching it? I think tslint is what's working in this repo for me, ATM. We need to unify this monorepo to use eslint.